packages/storefront-events-sdk/src/UnsubscribeManager.ts

```diff
@@ -43,6 +47,10 @@ export default class UnsubscribeManager extends Base {
         this.removeEventListener(events.OPEN_CART, handler);
     }
 
+    orderPageView(handler: EventHandler): void {
```

Comment:
Why are we adding order page view event?

Reply:
it is the context for the order page when the page is loaded. it was currently being set to requisition List but that doesn't make sense. the requisition list wouldn't can exist, but doesn't necessarily mean that the products of that order are on the list. once the "add to requisition list" button is hit, that is when the items should be added to the requisitionListContext

Reply:
removed